### PR TITLE
AFI Medical settings implementation

### PR DESCRIPTION
**Pull request by ZluskeN**
Baseline Medical Settings from AFI [Reference](https://armafinland.fi/forum/viewtopic.php?f=59&t=1896&start=40)

### CODE REVIEW

File: addons/userconfig/cba_settings.sqf [hunk 1]
```diff
@@ -16,18 +16,31 @@ force ace_finger_enabled = true;
 
 force ace_gforces_enabledFor = 0;
 
-force ace_goggles_effects = 0;
```
Comment (Dahlgren) [inline addons/userconfig/cba_settings.sqf:19]: Unrelated change

File: addons/userconfig/cba_settings.sqf [hunk 2]
```diff
@@ -16,18 +16,31 @@ force ace_finger_enabled = true;
...
-
 force ace_hearing_enableCombatDeafness = false;
 
 force ace_map_mapShowCursorCoordinates = true;
 
-force ace_medical_ai_enabledFor = 0;                      // Medical system not enabled for AI
-force ace_medical_feedback_painEffectType = 2;            // Pain effect type is "Chromatic Abberation".
-force ace_medical_gui_enableMedicalMenu = 1;              // Enable medical menu.
```
Comment (Dahlgren) [inline addons/userconfig/cba_settings.sqf:27]: Restore

File: addons/userconfig/cba_settings.sqf [hunk 3]
```diff
@@ -16,18 +16,31 @@ force ace_finger_enabled = true;
...
+force ace_medical_fatalDamageSource = 2;                  // Either large hits to vital organs or sum of trauma can be fatal
+force ace_medical_feedback_bloodVolumeEffectType = 2;     // Low blood volume effect is Icon + Color Fading
+force ace_medical_fractureChance = 0.3;                   // The probability of a fracture causing wound resulting in a fracture
+force ace_medical_ivFlowRate = 3;                         // IV Transfusion Flow Rate
+force ace_medical_painCoefficient = 1.4;                  // Coefficient for controlling the intensity of pain adjustments
+force ace_medical_playerDamageThreshold = 0.8;            // The amount of damage a player unit can take before dying.
+force ace_medical_spontaneousWakeUpChance = 0.15;         // Unconscious Wake Up Chance
+force ace_medical_spontaneousWakeUpEpinephrineBoost = 9;  // Epinephrine Wake Up Chance Boost
 force ace_medical_statemachine_AIUnconsciousness = false; // AI will not go unconscious for any reason, and therefore never enter cardiac arrest.
-force ace_medical_statemachine_fatalInjuriesAI = 0;       // AI will always die instantly.
```
Comment (Dahlgren) [inline addons/userconfig/cba_settings.sqf:30]: Add back key

File: addons/userconfig/cba_settings.sqf [hunk 4]
```diff
@@ -16,18 +16,31 @@ force ace_finger_enabled = true;
 
 force ace_gforces_enabledFor = 0;
 
-force ace_goggles_effects = 0;
-
 force ace_hearing_enableCombatDeafness = false;
 
 force ace_map_mapShowCursorCoordinates = true;
 
-force ace_medical_ai_enabledFor = 0;                      // Medical system not enabled for AI
```
Comment (Dahlgren) [inline addons/userconfig/cba_settings.sqf:25]: Add back key

File: addons/userconfig/cba_settings.sqf [hunk 5]
```diff
@@ -16,18 +16,31 @@ force ace_finger_enabled = true;
...
-force ace_medical_feedback_painEffectType = 2;            // Pain effect type is "Chromatic Abberation".
```
Comment (Dahlgren) [inline addons/userconfig/cba_settings.sqf:26]: Add back key

File: addons/userconfig/cba_settings.sqf (hunk 1 above)
Comment (ZluskeN) [reply addons/userconfig/cba_settings.sqf:19]: No need to force this settings server-side, players should be able to use goggle effects if they want to

File: addons/userconfig/cba_settings.sqf (hunk 5 above)
Comment (ZluskeN) [reply addons/userconfig/cba_settings.sqf:26]: there have been reports of chromatic aberration acting up, 
for now we should leave this undefined and let players decide what to use

File: addons/userconfig/cba_settings.sqf (hunk 4 above)
Comment (ZluskeN) [reply addons/userconfig/cba_settings.sqf:25]: This should be set to 1, which is already default value, thus no need to set it here

File: addons/userconfig/cba_settings.sqf (hunk 3 above)
Comment (ZluskeN) [reply addons/userconfig/cba_settings.sqf:30]: This is the default value, no need to define it here

File: addons/userconfig/cba_settings.sqf (hunk 5 above)
Comment (Dahlgren) [reply addons/userconfig/cba_settings.sqf:26]: Either we have "Chromatic Abberation" or we use the other one as default

File: addons/userconfig/cba_settings.sqf (hunk 2 above)
Comment (ZluskeN) [reply addons/userconfig/cba_settings.sqf:27]: Displaying medical menu is a player-side preference, no need to force it

Comment (Dahlgren) [reply addons/userconfig/cba_settings.sqf:27]: Players forget to enable it, therefor it's nice to have it as an option out of the box

File: addons/userconfig/cba_settings.sqf (hunk 3 above)
Comment (Dahlgren) [reply addons/userconfig/cba_settings.sqf:30]: It's good to have for when someone decies to change it :)

And as reference is someone wants to look at our options

File: addons/userconfig/cba_settings.sqf (hunk 4 above)
Comment (Dahlgren) [reply addons/userconfig/cba_settings.sqf:25]: It's good to have for when someone decies to change it :)

And as reference is someone wants to look at our options

File: addons/userconfig/cba_settings.sqf (hunk 1 above)
Comment (Dahlgren) [reply addons/userconfig/cba_settings.sqf:19]: Not related to ACE medical changes

File: addons/userconfig/cba_settings.sqf (hunk 5 above)
Comment (ZluskeN) [reply addons/userconfig/cba_settings.sqf:26]: Let's try the new "Pulsing Blur"

File: addons/userconfig/cba_settings.sqf [hunk 6]
```diff
@@ -22,12 +22,56 @@ force ace_hearing_enableCombatDeafness = false;
...
+force ace_medical_bleedingCoefficient = 1.2;              // Coefficient for controlling the bleeding speed
+force ace_medical_blood_bloodLifetime = 900;              // Lifetime of blood drop objects
+force ace_medical_blood_enabledFor = 2;                   // Blood drops enabled for all units
+force ace_medical_blood_maxBloodObjects = 300;            // Maximum number of blood drop objects that can be spawned
+force ace_medical_fatalDamageSource = 2;                  // Either large hits to vital organs or sum of trauma can be fatal
+force ace_medical_feedback_bloodVolumeEffectType = 2;     // Low blood volume effect is Icon + Color Fading
+force ace_medical_feedback_painEffectType = 1;            // Pain effect type is "Pulsing Blur"
+force ace_medical_fractureChance = 0.3;                   // The probability of a fracture causing wound resulting in a fracture
+force ace_medical_fractures = 1;                          // Splints Fully heal fractures
+force ace_medical_gui_enableMedicalMenu = 2;              // Enable medical menu.
```
Comment (Dahlgren) [inline addons/userconfig/cba_settings.sqf:36]: `2` means `VehiclesOnly`, should be `1`?

Comment (ZluskeN) [reply addons/userconfig/cba_settings.sqf:36]: Dang, you're right. 2 is AFI setting, but let's keep it at 1
